### PR TITLE
[FLOW-2] Unbreak transcript import

### PR DESCRIPTION
**Pull request by dshynkev**
This is a clean slate reimplementation of transcript import that works with the new transcript format.

### CODE REVIEW

File: server/static/js/js_tests/transcript/data/basic.txt [hunk 1]
```diff
@@ -1,570 +1,107 @@
...
-
-
-
-
-
-
-
-
-Vimium has been updated to 1.39.x
+University of Waterloo
```
Comment (jlfwong) [inline server/static/js/js_tests/transcript/data/basic.txt]: Haha the replacement of these fixtures is wonderfully symbolic of the ownership transfer

File: server/static/js/js_tests/transcript/basic_test.js [hunk 2]
```diff
@@ -24,52 +24,52 @@ define(function(require) {
 
     testParsedTranscriptAsync('extracts the student number',
         function(transcriptData) {
-      expect(transcriptData.studentId).to.equal(20331374);
+      expect(transcriptData.studentId).to.equal(20705374);
```
Comment (AyushK1) [inline server/static/js/js_tests/transcript/basic_test.js]: I trust that this is a false student #

File: server/static/js/js_tests/transcript/data/basic.txt [hunk 3]
```diff
@@ -1,570 +1,107 @@
...
+Course Description Attempted Earned Grade
+CS 241E Foundations of Sequential Programs (Enriched) 0.50 0.50
+CS 245 Logic and Computation 0.50 0.50
+CS 246E Object-Oriented Software Development (Enriched) 0.50 0.50
+ECE 222 Digital Computers 0.50 0.50
+MATH 249 Introduction to Combinatorics (Advanced Level) 0.50 0.50
+In GPA Earned
+Term GPA 94.40 Term Totals 2.50 2.50
+Cumulative GPA 93.75 Cumulative Totals 8.00 8.00
+Major Average: 98.20 Computer Science/Digital Hardware Option, Honours, Co-operative Program
```
Comment (AyushK1) [inline server/static/js/js_tests/transcript/data/basic.txt]: 💪

**Comment by AyushK1** [comment]
👍

**Pull request merged**